src/lib/isRgbColor.js

```diff
@@ -1,9 +1,9 @@
 import assertString from './util/assertString';
 
-const rgbColor = /^rgb\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){2}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\)$/;
-const rgbaColor = /^rgba\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){3}(0?\.\d|1(\.0)?|0(\.0)?)\)$/;
-const rgbColorPercent = /^rgb\((([0-9]%|[1-9][0-9]%|100%),){2}([0-9]%|[1-9][0-9]%|100%)\)/;
-const rgbaColorPercent = /^rgba\((([0-9]%|[1-9][0-9]%|100%),){3}(0?\.\d|1(\.0)?|0(\.0)?)\)/;
+const rgbColor = /^rgb\(\s*(([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\s*,\s*){2}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\s*\)$/;
```

Comment:
I didn't test the regex yet but it looks like a potential reDoS attack.
Can you please refactor the way you are checking for spaces. It will be easier and more safe to strip spaces instead of checking for it inside each regex. As an example you can see how we are achieving the same purpose in [isHSL](https://github.com/validatorjs/validator.js/blob/c899b311319ce9e2f725d4601ef9019e077afbec/src/lib/isHSL.js#L12)

Reply:
alright, on it!